config/rulesets/solium-all.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
         "no-constant": "warning",
         "no-experimental": "warning",
         "max-len": "warning",
+        "modifier-order": "warning",
```

Comment:
Please change the rule's name to `visbility-first`. This seems much more intuitive :)